celestia/peers.txt

```diff
@@ -1 +1 @@
-
+deefd5d3a4d497be6772295d10174f021d344ac5@57.128.75.161:26656
```

Comment:
please fix conflict by submitting the peer info in a separate file in this directory named as `your-validator.txt`